tests/property/test_verify.py

```diff
@@ -111,3 +119,49 @@ def test_verify_decryption(self):
         # Assert
         self.assertIsNotNone(verification)
         self.assertTrue(verification.verified)
+
+    @settings(
+        deadline=timedelta(milliseconds=2000),
+        suppress_health_check=[HealthCheck.too_slow],
+        max_examples=10,
+        # disabling the "shrink" phase, because it runs very slowly
+        phases=[Phase.explicit, Phase.reuse, Phase.generate, Phase.target],
+    )
+    @given(integers(1, 3).flatmap(lambda n: elections_and_ballots(n)))
+    def test_verify_aggregation(self, election_details: ElectionsAndBallotsTupleType):
+        # Arrange
+        (
+            manifest,
+            _internal_manifest,
+            ballots,
+            _secret_key,
+            context,
+        ) = election_details
+        internal_manifest = InternalManifest(manifest)
```

Comment:
While refactoring I found out `_internal_manifest` and `internal_manifest` are not equal for some test examples. To pass this test I use `Manifest` and instead of `InternalManifest` returned by the function `elections_and_ballots()`.

Reply:
Created this issue: https://github.com/microsoft/electionguard-python/issues/501